lib/bitcoin/IBitcoinConfig.ts

```diff
@@ -11,6 +11,7 @@ export default interface IBitcoinConfig {
   bitcoinRpcUsername: string | undefined;
   bitcoinRpcPassword: string | undefined;
   bitcoinWalletOrImportString: IBitcoinWallet | string;
+  estimatedFeeInBitcoinsPerKB: number | undefined;
```

Comment:
I think it makes sense to be in satoshis so we are not dealing with small fraction numbers.
Also "default" is important because it conveys the message that this maybe overriden which is indeed the case in the implementation.
```suggestion
  defaultEstimatedFeePerKBInSatoshis: number | undefined;
```

Reply:
I agree on both accounts, but re using satoshis over bitcoins: `valueTimeLockTransactionFeesAmountInBitcoins` is in bitcoins so rather they stay in the same unit, no?

Reply:
Another reason why I think it makes sense to stick to bitcoin in the config file is that the RPC call this is referring to also returns in bitcoin. During ION setup you could use the RPC call to get the estimate and put it in the config file as is.

Reply:
I finally went with `defaultFeePerKBInSatoshis` which I think better describes what this ends up doing. "Default estimated fee" sounded too vague IMHO.